projects/igniteui-angular/src/lib/expansion-panel/expansion-panel-header.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="igx-expansion-panel__header-inner" tabindex="0" role="button" [attr.id]="id"
+[attr.aria-disabled]="disabled" [attr.aria-expanded]="isExpanded" [attr.aria-controls]="controls">
+    <div class="igx-expansion-panel__title-wrapper">
+        <ng-content select="igx-expansion-panel-title"></ng-content>
+        <ng-content select="igx-expansion-panel-description"></ng-content>
+    </div>
+    <!--<ng-content></ng-content>-->
```

Comment:
We should probably keep this one to allow general content to be placed in the header (media, etc). Maybe wrap it in a container so flex order can be applied on the custom content as a whole? 
I don't mind it being before the icon, though the `icon--end` class would be order 2 instead of 1 with the extra container.

My only concern is the extra container would be present regardless of content being provided. Maybe force it with a selector rather than container?